sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BatchLoads.java

```diff
@@ -189,7 +193,9 @@ public void getTempFilePrefix(ProcessContext c) {
                     new SimpleFunction<String, String>() {
                       @Override
                       public String apply(String input) {
-                        return stepUuid;
+                        String jobId = BigQueryHelpers.randomUUIDString();
+                        LOG.info("BigQuery export job id {}", jobId);
```

Comment:
I believe this is logged by the layer that actually sends the export job. Is there value to logging it twice? If not, should we just verify that layer logs it and then leave this out?